cirq/linalg/decompositions.py

```diff
@@ -168,10 +169,8 @@ def map_eigenvalues(
     Returns:
         The transformed matrix.
     """
-    vals, vecs = _perp_eigendecompose(matrix,
-                                      rtol=rtol,
-                                      atol=atol)
-    pieces = [np.outer(vec, np.conj(vec.T)) for vec in vecs]
+    vals, vecs = orthogonal_eigendecompose(matrix, rtol=rtol, atol=atol)
+    pieces = [np.outer(vec, np.conj(vec.T)) for vec in vecs.T]
```

Comment:
I think by switching from `vecs` to `vecs.T` you're also incidentally fixing  a bug. Thanks!

While you're at it, could you add a simple unit test to verify the behavior of  this function which would fail on previous code and succeed now? This will help us prevent regressions.

(This applies to `orthonormal_eigendecompose` above as well since you made a similar fix there, too.)

Reply:
There is a unit test, `test_orthogonal_eigendecompose`. (I would rename it to orthonormal)